src/classes/action/movie.vala

```diff
@@ -300,9 +327,11 @@ namespace pdfpc {
         public virtual void play() {
             if (this.eos) {
                 this.eos = false;
-                this.pipeline.seek_simple(Gst.Format.TIME, Gst.SeekFlags.FLUSH, 0);
+                this.pipeline.seek_simple(Gst.Format.TIME, Gst.SeekFlags.FLUSH, this.starttime * Gst.SECOND);
             }
+
             this.pipeline.set_state(Gst.State.PLAYING);
+
```

Comment:
There's no need for an empty line before a closing brace.  (The previous inserted line is also a pointless whitespace change, but one I won't bug you about.)